arboretum/ibm_cloud/README.md

```diff
@@ -18,10 +18,62 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster List
+
+* Class: [ClusterListFetcher][fetch-cluster-list]
+* Purpose: Write the list of IBM Cloud clusters to the evidence locker.
+* Behavior: Log in to IBM Cloud using `ibmcloud login` command, and save the result of `ibmcloud cs cluster ls` command.
+* Expected configuration elements:
+  * org.ibm_cloud.cluster_list.config
+    * List of objects representing the IKS accounts
+    * Each object must have the following values
+      * `account` - a list containing names identifying the IKS account, this will map to an IAM token provided in the credentials file
+* Expected configuration example:
```

Comment:
We changed the format a little to the READMEs.  ref: https://github.com/ComplianceAsCode/auditree-arboretum/blob/main/arboretum/auditree/README.md.  Update `Expected configuration elements` and `Expected configuration example` and anything else that seems different.

Reply:
applied changes to both [arboretum/ibm_cloud/README.md](https://github.com/ComplianceAsCode/auditree-arboretum/pull/15/files#diff-fa9c900ac18eaea9441dab042c0ad0c9)  and [arboretum/kubernetes/README.md](https://github.com/ComplianceAsCode/auditree-arboretum/pull/15/files#diff-f47957446fca3e1172a44cd6eda6abcc)